install/docker/cws-image/Dockerfile

```diff
@@ -0,0 +1,59 @@
+#FROM registry.access.redhat.com/rhscl/mariadb-101-rhel7:10.1-24
+#FROM registry.access.redhat.com/rhel:latest
+#FROM registry.access.redhat.com/rhel7
+FROM centos:7
+USER root
+RUN yum update -y && \
+yum install -y sudo which mariadb-server mariadb-client java-1.8.0-openjdk java-1.8.0-openjdk-devel ntp && \
```

Comment:
it seems that running ntp in containers is not a recommended practice.  suggest not installing ntp in container.